### PR TITLE
init multipool logic for liquidity, swap

### PR DESCRIPTION
**Pull request by tien7668**

### CODE REVIEW

File: src/pages/Swap/index_2.tsx [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export default function Swap2() {
```
Comment (piavgh) [inline src/pages/Swap/index_2.tsx:1]: This file is not necessary. We can remove it

File: src/constants/index.ts [hunk 2]
```diff
@@ -1,18 +1,545 @@
...
-export const ROUTER_ADDRESS = '0x8406Caa2Cc202aFB4eCfb066D472E462bee00f3b'
-export const FACTORY_ADDRESS = '0x945c725e3eCC3dfdC350C0334f3fF42f08F719EA'
-export const INIT_CODE_HASH = '0xf6eae63ebbc500de6e7310fc6568df4e6a4514aac0d3d423da5e4e3f332d04f5'
-export const BAD_RECIPIENT_ADDRESSES: string[] = [
-  FACTORY_ADDRESS,
-  ROUTER_ADDRESS
+export const ROUTER_ADDRESS = '0xF71Fe20e1a630d27b93FE3d17dFda4249F13cDcC'
+export const FACTORY_ADDRESS = '0xe25d045D9B65F5428B231f46BBA2B3F351A97F3b'
+export const ROUTER_ABI = [
```
Comment (andrewsource147) [inline src/constants/index.ts:8]: Can we generate ABI file, and move it out of the configuration?

File: src/pages/Swap/index_2.tsx (hunk 1 above)
Comment (tien7668) [reply src/pages/Swap/index_2.tsx]: agree

File: src/constants/index.ts (hunk 2 above)
Comment (tien7668) [reply src/constants/index.ts]: yes, done

**Pull request merged**